build.gradle

```diff
@@ -47,8 +35,29 @@ dependencies {
     compile gradleApi()
     compile localGroovy()
 
-    compile 'com.github.kongchen:swagger-maven-plugin:3.1.5'
+    compile 'commons-io:commons-io:2.6'
+    compile 'org.apache.commons:commons-lang3:3.6'
+    compile 'com.github.jknack:handlebars:4.0.6'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.9.2'
+    compile 'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.9.2'
+    compile 'javax.xml.bind:jaxb-api:2.2.4'
+    
+    //These should be pinned here until reflections/reflections#194 has been resolved
+    compile 'io.swagger:swagger-jaxrs:1.5.13'
+    compile 'io.swagger:swagger-jersey-jaxrs:1.5.13'
```

Comment:
After a bit of real world testing on some projects I quickly ran into https://github.com/ronmamo/reflections/issues/194 because the 0.9.11 version uses a method that doesn't exist on many versions of guava. I rolled these back to one that includes 0.9.10 which doesn't have this problem.